app/renderer/components/Inspector/shared.js

```diff
@@ -1,6 +1,26 @@
 import { DOMParser } from 'xmldom';
 import xpath from 'xpath';
 
+export function pixelsToPercentage (pixel, isX, width, height) {
+  if (!isNaN(pixel)) {
+    if (isX) {
+      return parseFloat(((pixel / width) * 100).toFixed(1), 10);
+    } else {
+      return parseFloat(((pixel / height) * 100).toFixed(1), 10);
+    }
+  }
+}
+
+export function percentageToPixels (percentage, isX, width, height) {
```

Comment:
given that we are now passing in width and height, i suggest the function would be more simply implemented as follows:

```js
function percentageToPixels (pct, maxPixels) {
  if (!isNaN(pct)) {
    return Math.round(maxPixels * (pct / 100));
  }
}
```

and then you just pass in `width` or `height` based on whether the pct is an x or y value. Same comment goes for `pixelsToPercentage`. also, are the percentages being passed in as 0-100? or 0-1? typically percentages would be represented in code as 0-1, in which case you don't need to do the `/ 100` step here

Reply:
The numbers being passed into the function are from 0-100, so we do need to `/ 100`! I decided it to do it this way just so its easier to display the values instead of having to multiply by 100 each time if we want to display the percentages